src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/custom.py

```diff
@@ -458,3 +472,48 @@ def return_valid_duration_create(update_value):
                 return None
     else:
         return None
+
+
+# NetwrokRuleSet Region
+def cli_networkrule_createupdate(client, resource_group_name, namespace_name, subnet=None, ip_mask=None, ignore_missing_vnet_service_endpoint=False, action='Allow'):
+    from azure.mgmt.servicebus.models import NWRuleSetVirtualNetworkRules, Subnet, NWRuleSetIpRules
+    netwrokruleset = client.get_network_rule_set(resource_group_name, namespace_name)
+
+    if netwrokruleset.virtual_network_rules is None:
+        netwrokruleset.virtual_network_rules = [NWRuleSetVirtualNetworkRules]
+
+    if netwrokruleset.ip_rules is None:
+        netwrokruleset.ip_rules = [NWRuleSetIpRules]
```

Comment:
Same. This should just be empty list `[]`. Otherwise you have a list that contains a single class, which seems like a major logic error.

Reply:
resolved